builtin/settingtypes.txt

```diff
@@ -2507,10 +2508,10 @@ keymap_toggle_chat (Chat toggle key) key KEY_F2
 #    Key for toggling the display of the large chat console.
 keymap_console (Large chat console key) key KEY_F10
 
-#    Key for toggling the display of fog.
-keymap_toggle_force_fog_off (Fog toggle key) key KEY_F3
+#    Key for toggling the display of fog. Only usable for debugging.
+keymap_toggle_fog (Fog toggle key) key KEY_F3
 
-#    Key for toggling the camera update. Only used for development
+#    Key for toggling the camera update. Only usable for debugging.
```

Comment:
Maybe mention the privilege? "debug option" (above) and "for debugging" (here) feel a bit vague.